### ISSUE TITLE
Higurashi Symlink Installer

### ISSUE THREAD

**Issue by P-Chang** [07th-mod/higurashi-patch-compiler#11]
Currently, outdated now.

**Comment by ItaloKnox** [comment]
Whoa, that's amazing, congrats. For now, I don't think we have a real use for it but it might save some time when we deal with future chapters.

Makes me want to make a patch updater with a GUI like that. Though command line is all I can manage for now.

**Comment by enumag** [comment]
I'm not quite sure what's the point. Previously there was a need to create these symlinks to save disk space but that's no longer the case.

**Comment by P-Chang** [comment]
@ItaloKnox 
Thank you! I think one that need it doesn't have big population, and it is not needed in soon for most of guys too.
If you need symlinks in later, it could be save your times.

@enumag 
It's just optional tool. there are option is always good in my opinion.

**for my use**

I've play mod since mod have installer and compact voice archive. Probably my purporse is different from old one.

Voice archives for each chapter are really compact in 2017 on internet or HDD. And these are really useful in first install.
but download speed really sometimes piss off me... it show sometimes nearly 50kb/s or lower. I want never update voice archives.

I'm bit a worrywart, so i always leave any mod backups. it means there are unziped voice in game folder, and zip file in backup disk.
and unziped and zip PS3 and PS2 voices. these took a some disk space. my eintire disk space is 6240GB but reaming space only 500GB. and my new motherboard just give me only 6 SATA... I need compress disk space.

I know full archive of ps3 and ps3 are bigger than entire separate archive, but full archive is needed in searching voice. so I use symlinks and make it for someone that same issue.

it just save few GB but it means I can install one more later 2000s 3D games lol


**for devs**

I think symlinks are still needed in developing. especially ps2 folders are completely different from ps3 folders.
Changing 06/*************.ogg to 01/s**/***************.ogg and copy it in gamefolder and write in scripts are bother works.

if you use this, write in scripts, and export with one click(next feature), finish. it is really simple.

and it doesn't have option for CG folder, but if we make lip-sync, we need symlinks again. every character sprite need 2 more files, and chapter 1 to 4 and chapter 5 to 9 CG folder could be combine.

**Comment by enumag** [comment]
Thanks for clarification. I probably won't use it myself for now but I agree it might be good in the future. Good job!

**Comment by P-Chang** [comment]
@enumag 
by the way, how do you think changing ps2 folders numbers?
Leaving same character folder are really complicated.

I prefer completely stick to ps3 folder, or use same as archive.

**Comment by enumag** [comment]
I'm not sure I understand the problem with ps2 vs ps3 directory numbers. I mean I improved the patches with quite a few ps2-only voices in the past but never noticed such issue. Probably I just completely ignored ps2 directory names and put the voices where they belonged according to ps3 names.

**Comment by P-Chang** [comment]
Hmm, grep detect 217 line "ps2" in Tata, and my wip exporter copied 204 files and no Teppei voice.
line have "ps2/s03/17"(Teppei in ps2) are 13. probably leaved character is only Teppei.

Problem is small than I thought.

Fortunately 17 are Oryo in ps3 and empty in renamed ps2 folder.

**Comment by enumag** [comment]
The problem is that previously I didn't put voices in PS2 to separate directory. I only started to do that in the middle of fixing Tatarigoroshi because I noticed that PS2 voices have a lower volume. To fix that I added a volume normalization script into my compiler which is directory based.

Other chapters and event the first few days of Tata don't use this convention.

Teppei had very few lines in Tata. If the directory number is wrong feel free to move his voices. Or tell me where they belong and I can do it for you.
